R/dashboards/descriptives03-app/app.R

```diff
@@ -23,58 +14,15 @@ library(shinycssloaders)
 library(bslib)
 library(mapview)
 library(stplanr)
+library(leafgl)
+library(colourvalues)
 
 options(shiny.reactlog = TRUE) # permits to launch reactlog
 mapviewOptions(fgb = F) # set to false for greater performance?
-#setwd("/Volumes/Al-Hakem-II/Scripts/bikeshare/R/dashboards/descriptives03-app")
 
 ui <- navbarPage("Bikeshare", # UI ===================================================
-  # tabPanel("Days", # page 1 -----------------------------------------------------
```

Comment:
I'm fine with removing the "Days" section here, because at this point that's not the point of the app anyway-- the point is to create the desire lines and map them.